doc/man3/OSSL_ENCODER_to_bio.pod

```diff
@@ -24,6 +27,21 @@ is undefined.
 
 =head1 DESCRIPTION
 
+OSSL_ENCODER_to_data() runs the encoding process for the context I<ctx>,
+with the output going to the I<*pdata> and I<*pdata_len>.
+If I<*pdata> is NULL when OSSL_ENCODER_to_data() is called, a buffer will be
+allocated using L<OPENSSL_zalloc(3)>, and I<*pdata> will be set to point at
+the start of that buffer, and I<*pdata_len> will be assigned its length when
+OSSL_ENCODER_to_data() returns.
+If I<*pdata> is non-NULL when OSSL_ENCODER_to_data() is called, I<*pdata_len>
+is assumed to have its size.  In this case, I<*pdata> will be set to point
+after the encoded bytes, and I<*pdata_len> will be assigned the number of
+remaining bytes.
+
+The application is required to set
+up the B<BIO> properly, for example to have it in text or binary mode if
+that's appropriate.
+
```

Comment:
This looks like a leftover.

Reply:
Not quite, but is was misplaced.  This is still applicable for OSSL_ENCODER_to_bio() and OSSL_ENCODER_to_fp().  I've moved it around to make better sense.

Reply:
Please note that it is there already.

Reply:
Ahhh, good point!  Okie, more editing